doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+Navigate to the file specified in the ``exe_args`` argument of the ``create_run_settings()``
+function, ``exe_args="/path/to/model_multidb_example.py"``.
+
+To begin, learn to retrieve data from the launched databases by using a SmartRedis Client
+and the database identifier. In the script, we do not have access to the
+``exp.get_address()`` function. To establish a connection with the launched databases,
+we still need to initialize a new SmartRedis client for each ``Orchestrator`` with the address of
+the launched database.
+
+Previously, we added the ``db_identifier`` argument when creating the two ``Orchestrators``.
+When a database is created with a ``db_identifier`` name, SmartSim creates an environment
+variable named ``SSDB`` for each database with the ``db_identifier`` variable value suffixed to
+``SSDB``. In the case of the example, the environment variables created are ``SSDB_single_shard_db_identifier``
+and ``SSDB_clustered_db_identifier``.
+
+To create a SmartRedis Client for each database, first use the
+``ConfigOptions.create_from_environment()`` factory method, which accepts the
+suffix to be applied to the ``SSDB`` environment variable. Then pass the ``ConfigOptions``
+object when creating the SmartRedis client.
+
+.. code-block:: python
+
+  from smartredis import ConfigOptions, Client
+  from smartsim.log import get_logger
+  import torch
+
+  logger = get_logger("Multidb App Log")
+
+For the standalone database:
+
+.. code-block:: python
+
+  colo_config = ConfigOptions.create_from_environment("single_shard_db_identifier")
+  colo_orch_client = Client(colo_config, "client_colo")
+
+For the clustered database:
+
+.. code-block:: python
+
+  clus_config = ConfigOptions.create_from_environment("clustered-db-identifier")
+  clus_orch_client = Client(clus_config, "client_clus")
+
+These steps allow us to set up SmartRedis clients for each ``Orchestrator`` and
+establish effective communication with the respective databases.
+
+To confirm a successful connection to each database, retrieve the tensors previously
+stored using the created SmartRedis clients. The ``get_tensor()`` Client method allows
+retrieval of a tensor by passing in the tensor name.
+
+.. code-block:: python
+
+  val1 = colo_orch_client.get_tensor("tensor_1")
+  val2 = clus_orch_client.get_tensor("tensor_2")
+  logger.info(f"The colocated db tensor is: {val1}")
+  logger.info(f"The clustered db tensor is: {val2}")
+
+When you execute the program the following output will appear in ``tutorial_mode.out``
+located in ``getting-started-multidb/tutorial_model/``.
+
+.. code-block:: bash
+
+  21:01:55 prod-0005 SmartSim[34075] INFO The colocated db tensor is: [1 2 3 4]
+  21:01:55 prod-0005 SmartSim[34075] INFO The clustered db tensor is: [5 6 7 8]
+
+This code showcases that we can retrieve data from each database effectively.
+
+Lastly, within the model, we will demonstrate creating and setting
```

Comment:
Is this strictly needed here? I think it's sufficient to end here and not add further information about adding a script to the database. There's nothing really different about performing this task with in an experiment with multiple databases